src/Tests/SNIPPETS.cs

```diff
@@ -97,42 +97,41 @@ @ EntryPoint (  )
         public static string OpenNamespaces1 =
 @"
     open Microsoft.Quantum.Intrinsic;
-    open Microsoft.Quantum.Measurement;
+    open Microsoft.Quantum.Diagnostics;
 ";
 
         public static string OpenNamespaces2 =
 @"
-    open Microsoft.Quantum.Canon;
-    open Microsoft.Quantum.Measurement;
+    open Tests.qss;
+    open Microsoft.Quantum.Diagnostics;
 ";
 
         public static string DependsOnMeasurementNamespace =
 @"
-    operation DependsOnMeasurementNamespace() : Result[]
+    operation DependsOnMeasurementNamespace() : Unit
```

Comment:
[nit] this and other references to `DependsOnMeasurementNamespace` can probably just become `DependsOnNamespace`, since it no longer actually depends on the `Measurement` namespace.